app/views/health_cards/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<%= javascript_pack_tag 'details_pg' %>
+<p id="notice"><%= notice %></p>
+<section class="hero is-link">
+  <div class="hero-body">
+    <p class="title">Patient Details</p>
```

Comment:
The button to get to this page is called "Health Card Details", but then this page is titled "Patient Details". We should make these consistent. I think it makes sense to title this page "Health Card Details", Since it is showing how the actual health card was built up, not the patient resource (the patient show page where the button lives would technically be the "Patient Details" page).